src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -671,12 +670,6 @@ public void onOpen(@NotNull IChannel channel) {
     }
 
     public void sendHeartbeat() {
-        long secondsSinceLastSend = (System.currentTimeMillis() - lastSendTime) / 1000;
-
-        if (secondsSinceLastSend < settings.getHeartBtInt()) {
-            return;
-        }
```

Comment:
Why this was removed?

Reply:
It is no longer needed with current approach: we already reset heartbeatTask after every send